packages/orbit-components/src/tmp-tailwind/SegmentedSwitch/SwitchSegment/index.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from "react";
+import cx from "clsx";
+
+import useClickOutside from "../../../hooks/useClickOutside";
+import type { Props } from "../../../SegmentedSwitch/SwitchSegment/types";
+
+const SwitchSegment = ({
+  value,
+  label,
+  disabled,
+  onChange,
+  setTooltipShown,
+  onFocus,
+  defaultChecked,
+  name,
+}: Props) => {
+  const ref = React.useRef<HTMLInputElement | null>(null);
+  useClickOutside(ref, () => setTooltipShown(false));
+
+  return (
+    <label className="orbit-switch-segment-label relative flex w-full">
+      <input
+        className={cx(
+          "absolute m-[-1px] h-[1px] w-[1px] appearance-none overflow-hidden whitespace-nowrap border-0 p-0 focus:outline-none",
```

Comment:
I think you forgot here *rect* styles

It's a pattern, to have it available for a11y (screen readers). We use it on multiple places in Orbit 

I think you can apply just [this class](https://github.com/tailwindlabs/tailwindcss/discussions/6143) and [in tailwind docs](https://tailwindcss.com/docs/screen-readers#screen-reader-only-elements)

Reply:
I didn't know this class existed, thanks!

I had replaced it with `appearance-none`, but as you said, it might have some a11y implications.